console/frontend/src/main/frontend/src/app/app.component.ts

```diff
@@ -332,28 +340,30 @@ export class AppComponent implements OnInit, OnDestroy {
         });
       }
     });
-    this.appService.getClusterMembers().subscribe((data) => {
-      this.clusterMembers = data;
-    });
   }
 
   initializeWebsocket(): void {
-    this.websocketService.onConnected$.subscribe(() => {
-      this.appService.updateLoading(false);
-      this.loading = false;
+    this.appService.updateLoading(false);
+    if (this.loading) {
+      this.websocketService.onConnected$.subscribe(() => {
+        this.loading = false;
 
-      this.websocketService.subscribe<Record<string, MessageLog>>('/event/server-warnings', (configurations) =>
-        this.processWarnings(configurations),
-      );
+        const channelBaseUrl = this.selectedClusterMember ? `/event/${this.selectedClusterMember.id}` : '/event';
 
-      this.websocketService.subscribe<Record<string, Partial<Adapter>>>('/event/adapters', (adapters) =>
-        this.processAdapters(adapters),
-      );
+        this.websocketService.subscribe<Record<string, MessageLog>>(
+          `${channelBaseUrl}/server-warnings`,
+          (configurations) => this.processWarnings(configurations),
+        );
```

Comment:
Ik heb https://github.com/frankframework/frankframework/issues/7418 aangemaakt omdat hier een foutje in zit.

Reply:
Opgelost in de nieuwe commit (https://github.com/frankframework/frankframework/pull/7411/commits/fef4555a0ee4c9c22f999230fe825ccf973a78e5)